linux/net/rina/ipcps/shim-eth-vlan.c

```diff
@@ -51,6 +51,8 @@
 #define DEFAULT_QDISC_MAX_SIZE 50
 #define DEFAULT_QDISC_ENABLE_SIZE 10
 
+#define BLOCKING_FLOW 1
```

Comment:
It's just for readability. if you prefer, I can just put a 
```
bool true /* blocking flow */
```
in the parameter list for the flow allocation. I thing that at some point, there will be be some OPTIONS in the API such as "FLOW_OPT_BLOCKING" and then it will be obvious to replace this in the code.